pkg/assembler/graphql/resolvers/package.resolvers.go

```diff
@@ -6,29 +6,39 @@ package resolvers
 
 import (
 	"context"
+	"fmt"
 
+	"github.com/guacsec/guac/pkg/assembler/graphql/helpers"
 	"github.com/guacsec/guac/pkg/assembler/graphql/model"
 )
 
 // IngestPackage is the resolver for the ingestPackage field.
-func (r *mutationResolver) IngestPackage(ctx context.Context, pkg model.PkgInputSpec) (string, error) {
+func (r *mutationResolver) IngestPackage(ctx context.Context, pkg model.PkgInputSpec) (*model.PackageIDs, error) {
+	// Return the id of the PackageVersion which has been ingested
+	// TODO (knrc) - check why this was originally returning the ID for the Package, since that is not specific enough
```

Comment:
Returning only IDs was changed because of this issue: https://github.com/guacsec/guac/issues/1116

Reply:
+1, I was wondering specifically why the ID of the outer Package was returned.  I couldn't think of any useful purpose for that ID but wasn't sure if I was missing something.

Reply:
yeah, that was a mistake that we did not catch. Originally it should have been the pkg version ID.

Reply:
Note in the latest commit I've updated the Source ingestion to return multiple IDs, as with package ingestion